internal/model/storage.go

```diff
@@ -61,6 +61,13 @@ type DiskUsage struct {
 	FreeSpace  uint64 `json:"free_space"`
 }
 
+func NewDiskUsageFromUsedAndTotal(used, total uint64) *DiskUsage {
+	return &DiskUsage{
+		TotalSpace: max(used, total),
```

Comment:
There are three reasonable ways to address this:

1. Total = max(used, total), Free = total - min(used, total); if used > total, total will show used size. (e.g. 0/25TB, but if user deletes some file, it may become 0/24TB)
2. Total = total, Free = total - min(used, total); if used > total, free will be zero and total will be "allowed size", less than used size. (e.g. 0/20TB, but actually have 25TB data in storage)
3. Total = total, Free(change to int64) = int64(total) - int64(used); if used > total, free will be negative (e.g. -1/20TB)